### PR TITLE
Go 1.19

### PR DESCRIPTION
**Pull request by liran-funaro**
Signed-off-by: Liran Funaro <liran.funaro@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -23,3 +23,52 @@ require (
...
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/magiconair/properties v1.8.1 // indirect
+	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
+	github.com/mitchellh/mapstructure v1.1.2 // indirect
+	github.com/pelletier/go-toml v1.4.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/prometheus/client_golang v1.0.0 // indirect
+	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4 // indirect
+	github.com/prometheus/common v0.4.1 // indirect
+	github.com/prometheus/procfs v0.0.2 // indirect
```
Comment (cendhu) [inline go.mod:54]: I assume these dependencies come from uncommitted PR. Either make this PR independent or keep the dependency chain by including all commits in this PR.

Comment (liran-funaro) [reply go.mod:54]: This is not due to uncommitted code.
Since Go version 1.17, `go mod tidy` adds these **indirect** dependencies.

Comment (cendhu) [reply go.mod:54]: I see. Okay.

**Review by cendhu — APPROVED** [review]

**Comment by tock-ibm** [comment]
@liran-funaro we need to do this on the SDK as well

**Comment by liran-funaro** [comment]
@tock-ibm I already submitted a PR for this yesterday.

**Pull request merged**